### DISCUSSION TITLE
pathlib and hvplot not "accessed"

### DISCUSSION THREAD

**Discussion by ellenalamont17** [earthlab-education/Earth-Analytics-2023-01-Intro#359] [Q&A04 Raster Data]
I am having trouble importing two of the packages:

import pathlib 
import hvplot.pandas

I've spent some time poking around online, but most of the error solutions provided there seem larger and not necessarily applicable. I've tried a few options, but I just does not seem to import. Both provide an error comment stating each is not "accessed". I had no issues with hvplot in the first exercise, so I am unsure of the issue now. Is there maybe something wrong with my settings or environment? Thanks for your suggestions!

**Comment by jsmi374** [comment]
Hello Ellen, 

You might have the same issue I was experiencing. Attached is a figure showing my import code. You may need to split up the cells as shown. Let me know if this helps or not. 

@ellenalamont17 

![import_split](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/76793232/03798e8c-8ef8-48de-9352-54b45a8f26df)

**Reply by eculler** [reply]
@jsmi374 interesting - you should not need to split up the imports. You should also not need to specify the bokeh extension, as that is the default. Are you also getting the same error when you don't do those things?

**Comment by eculler** [comment]
@ellenalamont17 I'm not sure what's going on here. There was, however, a problem with the environment for this assignment that was not fixed until Tuesday evening. If you accepted the assignment and started your Codespace before then, please try starting a new Codespace (you can delete the old one once you have any changes you want out of it.)

If that doesn't work, I need a little more information to investigate this. In particular, the full error traceback as text or a screenshot would be useful. Are you getting that error for `import pathlib` alone? If so, the problem goes beyond the environment because `pathlib` is in the standard library and should always be able to import.

**Reply by ellenalamont17** [reply]
@eculler - I went in and made a new codespace. The following is a screenshot of the import cell still showing errors:

![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/143370562/1f4cf4b0-d8b0-4eeb-abf8-e0d4bd5d8cca)

Both the pathlib and the hvplot are greyed out (muted orange) in the cell. When I hover over pathlib, I get a message stating "Pathlib is not accessed pylance." I get the same message for hvplot. When I run the test cell, it states:

![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/143370562/87225d40-8bdc-4c4f-90e3-28c93f5f7840)

There is no error from the testcode for pathlib specifically, just the muted text.

**Comment by misterskye** [comment]
For what ever reason, I'm having an issue importing hvplot.pandas as well and yes I have just started a new Codespace.

Thanks,
skye

**Comment by misterskye** [comment]
Hey @ellenalamont17,

Wondering if you were ever able to get hvplot working.  I was in the field all of last week and trying to get caught up.

Thanks,
skye

**Reply by ellenalamont17** [reply]
@misterskye I was able to get it to work eventually. I had to commit and sync my changes to my ipynb file. Then I created a new codespace, checked that the ipynb loaded okay into the new codespace, and deleted the old one. That seemed to take care of my issue. However, when I was creating my personal portfolio analysis, I had to update my personal github page dev container to match the homework dev container before building a new codespace. Otherwise the appeears module will not load when you try to recreate the assignment.
